src/components/videoPage/VideoAIMetadataContent.tsx

```diff
@@ -0,0 +1,50 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import s from 'src/components/videoPage/style.module.less';
+import Badge from '@boclips-ui/badge';
+import { Typography } from '@boclips-ui/typography';
+import React from 'react';
+
+interface Props {
+  isLoading: boolean;
+  metadata?: string[];
+  type: string;
+}
+export const VideoAIMetadataContent = ({
+  isLoading,
+  metadata,
+  type,
+}: Props) => {
+  const getAIBadge = () =>
```

Comment:
getAiBadge, ShowContent, showErrorMessage could be components, that would improve readability the render method